README.rst

```diff
@@ -51,15 +46,68 @@ NeMo's works with:
 1) Python 3.6 or 3.7
 2) Pytorch 1.6 or above
 
-Installation
-~~~~~~~~~~~~
-``pip install nemo_toolkit[all]==version``
-
-We recommend using NVIDIA's PyTorch container
+Docker container:
+~~~~~~~~~~~~~~~~~
+We recommend using NVIDIA's PyTorch container version 20.06-py3 with NeMo's main branch.
 
 .. code-block:: bash
 
     docker run --gpus all -it --rm -v <nemo_github_folder>:/NeMo --shm-size=8g \
     -p 8888:8888 -p 6006:6006 --ulimit memlock=-1 --ulimit \
     stack=67108864 nvcr.io/nvidia/pytorch:20.06-py3
 
+Installation
+~~~~~~~~~~~~
+Once requirements are satisfied (or you are inside NVIDIA docker container), simply install using pip:
+
+``pip install nemo_toolkit[all]==version``
+
+* ``pip install nemo_toolkit[all]`` - latest released version (currently 0.11.0)
+* ``python -m pip install git+https://github.com/NVIDIA/NeMo.git@{BRANCH}#egg=nemo_toolkit[nlp]`` - where {BRANCH} should be replaced with the branch you want. This is recommended route if you are testing out the latest WIP version of NeMo.
+
+
+Documentation
+-------------
+* `Main <https://docs.nvidia.com/deeplearning/nemo/developer_guide/en/candidate/>`_
+* `Latest released (0.11.0) <https://docs.nvidia.com/deeplearning/nemo/developer_guide/en/v0.11.0/>`_
+
+Tutorials
+---------
+The best way to get started with NeMo is to checkout one of our tutorials.
+
+Most NeMo tutorials can be run on `Google's Collab <https://colab.research.google.com/notebooks/intro.ipynb>`_.
+
+Steps to run tutorial:
+
+1. Go to `Google's Collab <https://colab.research.google.com/notebooks/intro.ipynb>`_.
+2. Import tutorial notebook from GitHub (File -> Upload Notebook -> "GITHUB" tab -> copy/paste GitHub URL)
```

Comment:
Btw, we can later add a badge to all of our notebooks to directly point to colab instead of users having to do this manually, it should reduce friction even more.